app/components/auk/datepicker.hbs

```diff
@@ -7,15 +7,18 @@
     @allowInput={{true}}
     @clickOpens={{true}}
     @date={{this.date}} {{!-- Required Option --}}
+    {{!-- @dateFormat={{if @enableTime "d-m-Y H:i" "d-m-Y"}} --}}
```

Comment:
@erikap Deze arguments had ik al toegevoegd om de time input CSS te kunnen testen, maar heb ze uiteindelijk weer in commentaar geplaatst zodat ze nog niet per ongeluk gebruikt kunnen worden (voor de story mbt time input support aan bod komt).